src/test/java/uk/gov/pay/card/bench/RangeSetCardInformationStoreBenchmark.java

```diff
@@ -24,6 +24,18 @@
  * - Change the name of the file to load in this benchmark (in the setup()) method.
  * - You would have to replace the existing test card prefixes with some new prefixes.
  * - And then run the benchmark.
+ * 
+ * The Benchmark outputs something like the following:
+ * 
+ * Benchmark                                               (cardIdPrefix)  Mode  Cnt  Score   Error  Units
+ * RangeSetCardInformationStoreBenchmark.runBenchmark  511226661120000000  avgt   20  0.145 ± 0.001  us/op
+ * RangeSetCardInformationStoreBenchmark.runBenchmark  511226661130000000  avgt   20  0.142 ± 0.004  us/op
+ * RangeSetCardInformationStoreBenchmark.runBenchmark  511226661140000000  avgt   20  0.146 ± 0.002  us/op
+ * etc
+ * 
+ * This means the test card 511226661120000000 took 0.145 microseconds to run on average over 20 iterations.
+ * Assuming that the results are normally distributed, one would expect that the "true" execution time for 
+ * that method has a 99.9% probability to be somewhere between 0.145± 0.001 microseconds.
```

Comment:
Usually the error represents the range you'd expect for 68% of the results to land in (when you assume normally distributed), though not sure if this library is different for some reason

Reply:
Do you have a doc link for this? I found https://openjdk.org/projects/code-tools/jmh/ rather opaque, and gathered my info from https://stackoverflow.com/questions/33604084/understanding-jmh-output.

Reply:
Ignore me I had a look at how it's implemented: https://github.com/openjdk/jmh/blob/6d6ce6315dc39d1d3abd0e3ac9eca9c38f767112/jmh-core/src/main/java/org/openjdk/jmh/results/Result.java#L148

They've chosen to present their error confidence intervals at 99.9%.